website/server/libs/user/index.js

```diff
@@ -111,7 +111,25 @@ export async function update (req, res, { isV3 = false }) {
     const newName = req.body['profile.name'];
     if (newName === null) throw new BadRequest(res.t('invalidReqParams'));
     if (newName.length > 30) throw new BadRequest(res.t('displaynameIssueLength'));
-    if (nameContainsSlur(newName)) throw new BadRequest(res.t('displaynameIssueSlur'));
+
+    if (stringContainsSlur(newName)) {
```

Comment:
can this and the next if be combined into one since they have the same code?

```
if (stringContainsSlur(newName) || stringContainsBannedWord(newName))
```